app/styles/components/_grid.scss

```diff
@@ -1,11 +1,26 @@
 $grid-max-width: 1200px;
+$grid-break-mobile: 1049px;
 
 .Grid {
   display: flex;
   flex-direction: row;
   flex-wrap: wrap;
-  justify-content: space-between;
+  justify-content: center;
 
   max-width: $grid-max-width;
   margin-top: $Theme-margin-half;
 }
+
+.Grid--marginRightDesktop {
```

Comment:
I forgot the size part of the name so it should be Grid--marginRightDefaultDesktop. This name probably should be improved as well as the ones I chose for the margin utilities